specification/resources/resource-manager/Microsoft.Resources/2015-11-01/resources.json

```diff
@@ -2799,6 +2799,10 @@
             "$ref": "#/definitions/ResourceProviderOperationDefinition"
           },
           "description": "Gets or sets the list of resource provider operations."
+        },
+        "nextLink": {
```

Comment:
Example of this property being added exactly like this in a later API Version:

https://github.com/Azure/azure-rest-api-specs/blob/7c6c49a6d8507db1b70909b6b380d7f3bd4168cc/specification/resources/resource-manager/Microsoft.Resources/2016-07-01/resources.json#L2378